2016/proposals/narayan-aditya-visualizer.md

```diff
@@ -0,0 +1,244 @@
+# Survey Visualizer Application
+
+## Abstract
+
+Software Carpentry organizes workshops to teach computing skills and
+tools to researchers. The attendees and the organizers of the
+workshops fill in different surveys to allow the organization hosts
+to gather insight into the attendees computing background while
+the administrators collect long-term feedback.
+These surveys, which are hosted at [SurveyMonkey], are being
+analyzed using the SurveyMonkey web dashboard. Filtering the responses
+by a particular workshop can be cumbersome for the organization hosts
+and administrators as it requires a lot of clicks.
+
+The aim of this project is to build a web application to collect
+and visualize survey responses for every workshop, while providing
+a better experience for the administrators and organization hosts.
+
+## Technical Details
+
+The *Survey Visualizer* application stack can be divided into:
+
+ * Backend  : [Django Rest Framework]
+ * Scheduler: [Cron]
+ * Database : [PostgreSQL]
+ * Frontend : [Charts.js] and [Bootstrap]
+
+### Backend - Django Application
+
+The survey visualizer application would be made using
+[Django Rest Framework] (henceforth called DRF) over [Django]
+to allow easier integration with [AMY] when required.
```

Comment:
I will translate the above REST API to a corresponding python API for the pluggable app.
The way to serve these survey responses must be the task of the django project and not the app.